modules/apps/transfer/keeper/grpc_query.go

```diff
@@ -22,17 +23,21 @@ func (q Keeper) DenomTrace(c context.Context, req *types.QueryDenomTraceRequest)
 		return nil, status.Error(codes.InvalidArgument, "empty request")
 	}
 
-	hash, err := types.ParseHexHash(req.Hash)
+	if !strings.HasPrefix(req.Denom, "ibc/") {
+		return nil, status.Error(codes.InvalidArgument, fmt.Sprintf("invalid denom with no ibc prefix: %s", req.Denom))
+	}
```

Comment:
I think we should make this change backwards compatible. We could do so by checking if it has the ibc prefix and then remove the prefix before performing the query. I should have specified this on the original issue (but at the time there was a lot less dependencies on this gRPC

So essentially:
```go
if strings.HasPrefix(denom, "ibc/") {
    denom = denom[4:]
}

Reply:
So I'll make it like you said and leave the field name still be `denom`

Reply:
That's a good suggestion from @colin-axner. If you make the change backwards compatible, then the changelog entry shouldn't be in the API breaking section, but in the improvements one instead.

Reply:
I'll apply the change accordingly